tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheStats.java

```diff
@@ -78,9 +78,9 @@ public static void registerCache(MetricRegistry registry, Cache<?, ?> cache, Str
      * @param cache cache to instrument
      * @param name cache name
      * <p>
-     * Soon to be deprecated, prefer {@link Caffeine#recordStats(Supplier)} and {@link CacheStats#of(TaggedMetricRegistry, String)}
+     * @deprecated Prefer {@link Caffeine#recordStats(Supplier)} and {@link CacheStats#of(TaggedMetricRegistry, String)}
```

Comment:
While we're here, can we add a link to the readme and update the JavaDoc here (maybe embed the recommended examples from readme)?

```suggestion
     * @deprecated Prefer {@link CacheStats#of(TaggedMetricRegistry, String)}.{@link CacheStats#register(Function) register(stats ->) {@link Caffeine#recordStats(Supplier) Caffeine.newBuilder().recordStats(stats)}...  
     * <p/>
     * See <a href="https://github.com/palantir/tritium?tab=readme-ov-file#instrumenting-a-caffeine-cache">Instrumenting a [Caffeine cache](https://github.com/ben-manes/caffeine/)</a>
```